pattern-1/bosh-release/jobs/wso2is_2/templates/ctl.erb

```diff
@@ -0,0 +1,112 @@
+#!/bin/bash
+# ----------------------------------------------------------------------------
+#
+# Copyright (c) 2017, WSO2 Inc. (http://www.wso2.org) All Rights Reserved.
+#
+# WSO2 Inc. licenses this file to you under the Apache License,
+# Version 2.0 (the "License"); you may not use this file except
+# in compliance with the License.
+# You may obtain a copy of the License at
+#
+# http://www.apache.org/licenses/LICENSE-2.0
+
+# Unless required by applicable law or agreed to in writing,
+# software distributed under the License is distributed on an
+# "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+# KIND, either express or implied.  See the License for the
+# specific language governing permissions and limitations
+# under the License.
+#
+# ----------------------------------------------------------------------------
+
+set +e
+set -xv
+
+SCRIPT_DIR=$(dirname $0  )
+JOB_DIR=$SCRIPT_DIR/..
+
+RUN_DIR=/var/vcap/sys/run/wso2is
+LOG_DIR=/var/vcap/sys/log/wso2is
+PIDFILE=${RUN_DIR}/pid
+
+mkdir -p $RUN_DIR $LOG_DIR
+chown -R vcap:vcap $RUN_DIR $LOG_DIR
+
+mkdir -p /var/vcap/store/wso2is/data
+export WSO2_IS_DATA_DIR=/var/vcap/store/wso2is/data/wso2is
```

Comment:
What would be the usage of WSO2_IS_DATA_DIR?

Reply:
@imesh after tests, the usage of `WSO2_IS_DATA_DIR` was identified as unnecessary. These sections are available here, as I have reused the existing implementation. I have refined the control scripts and rest of the implementation as part of improvements and will be available in future PRs.